Polyhedron/demo/Polyhedron/Scene_c3t3_item.cpp

```diff
@@ -1497,7 +1497,18 @@ void Scene_c3t3_item_priv::computeSpheres()
     if(red)
       c = QColor(Qt::red);
     else
-      c = spheres->color().darker(250);
+      c = QColor::fromHsv(120, 200,200,255);
```

Comment:
The sphere color has one too, and whatever function we use, if we want the sphere to change color the usual way, we have to compute the whole of them each time (multicolor items require that).